pytype/pyc/pyc.py

```diff
@@ -69,17 +70,23 @@ def compile_src_string_to_pyc_string(src, filename, python_version, python_exe,
     # In order to be able to compile pyc files for both Python 2 and Python 3,
     # we spawn an external process.
     if python_exe:
-      # Allow python_exe to contain parameters (E.g. "-T")
-      exe = python_exe.split()
+      if sys.platform == 'win32':
+        exe = ["py"]
```

Comment:
nit: you can combine this line and the next one as:
`exe = ["py", "-" + ".".join(map(str, python_version))]`